paparazzi/src/main/java/app/cash/paparazzi/internal/ImageUtils.kt

```diff
@@ -114,33 +114,45 @@ internal object ImageUtils {
     }
     assertEquals(TYPE_INT_ARGB.toLong(), goldenImage.type.toLong())
 
-    val imageWidth = Math.min(goldenImage.width, image.width)
-    val imageHeight = Math.min(goldenImage.height, image.height)
+    val goldenImageWidth = goldenImage.width
+    val goldenImageHeight = goldenImage.height
+
+    val imageWidth = Math.max(goldenImageWidth, image.width)
+    val imageHeight = Math.max(goldenImageHeight, image.height)
 
     // Blur the images to account for the scenarios where there are pixel
     // differences
     // in where a sharp edge occurs
     // goldenImage = blur(goldenImage, 6);
     // image = blur(image, 6);
-
-    val width = 3 * imageWidth
+    val width = goldenImageWidth + imageWidth + image.width
     val deltaImage = BufferedImage(width, imageHeight, TYPE_INT_ARGB)
     val g = deltaImage.graphics
 
     // Compute delta map
     var delta: Long = 0
     for (y in 0 until imageHeight) {
       for (x in 0 until imageWidth) {
-        val goldenRgb = goldenImage.getRGB(x, y)
-        val rgb = image.getRGB(x, y)
+        val goldenRgb = if (x >= goldenImageWidth || y >= goldenImageHeight) {
+          0x00808080
+        } else {
+          goldenImage.getRGB(x, y)
+        }
+
+        val rgb = if (x >= image.width || y >= image.height) {
```

Comment:
Is there a reason for treating `image.width` and `goldenImageWidth` differently? I mean the fact that one is cached in a `val` and the other isn't.

Reply:
Yep, I mean that you could
* either inline `goldenImageWidth`
* or extract an `actualImageWidth` variable

It looks mixed up with using both approaches for very similar things.

Reply:
Ah gotcha, yeah that's a good point, the only reason goldenImageWidth and goldenImageHeight are in a val is because they are also used in other calculations outside the loop, but definitely makes sense to be consistent, will update this